.github/workflows/erlang-checks.yml

```
@@ -30,10 +30,13 @@ jobs:
   run:
     name: Run checks
     needs: setup
-    uses: valitydev/erlang-workflows/.github/workflows/erlang-parallel-build.yml@v1.0.12
+    uses: valitydev/erlang-workflows/.github/workflows/erlang-parallel-build.yml@v1.0.18
     with:
       otp-version: ${{ needs.setup.outputs.otp-version }}
       rebar-version: ${{ needs.setup.outputs.rebar-version }}
       use-thrift: true
       thrift-version: ${{ needs.setup.outputs.thrift-version }}
       run-ct-with-compose: true
+      use-coveralls: true
```

Мы вроде от всех них отказались уже, нет
```suggestion
      use-coveralls: false
```